website/static/js/project.js

```diff
@@ -336,7 +336,7 @@ $(document).ready(function() {
 
         // Remove Comments link from project nav bar for pages not bound to the comment view model
         var commentsLinkElm = document.getElementById('commentsLink');
-        if (!ko.dataFor(commentsLinkElm)) {
+        if (!ko.dataFor(commentsLinkElm) && commentsLinkElm != null) {
```

Comment:
This should NOT be here, this is a lingering change from another ticket I was on. I didn't pull from develop before I started working on this ticket let me see if I can decommit these two changes.